### DISCUSSION TITLE
Use in Unity

### DISCUSSION THREAD

**Discussion by guischutzer** [rajanadar/VaultSharp#178] [Q&A] [answered]
How do you recommend we use it on Unity? I was thinking of just using the source code, but that has a lot of problems.

**Comment by rajanadar** [comment, marked as answer]
Can you please expand on your question? The language you are using, your use case, do you use other libraries of any languages etc.

**Reply by guischutzer** [reply]
Yes! Of course.

Unity uses C# but doesn't have integrated NuGet support. I'm integrating VaultSharp to use Vault on my C# application, so that's not a problem. I'm more concerned with ease of updates and maintenance.

**Reply by rajanadar** [reply]
Got it.

If unity does not support NuGet, does it support plain library references? If yes, you should know that I also release a NuGet equivalent library as part of github releases as well. 
https://github.com/rajanadar/VaultSharp/releases 

It is always in sync with the NuGet releases and can be used in your case. Of course NuGet manages specific dot net versions for you. In this case, you have to select the right library from the release, as per your dot net platform. 

Let me know if that helps.

**Reply by guischutzer** [reply]
That's perfect, thanks!
